website/src/main/tut/docs/users/configuration.md

```diff
@@ -121,32 +120,72 @@ debug.printSymbols = true
 
 To build custom rules see {% doc_ref Rule Authors %}.
 
-## Per source file
 
-It's possible to supress false positives linter message over source file regions using comments. This provides fine grained control over what to report
+## Suppressing rules
```

Comment:
Great writing! You have a knack for docs

Reply:
I just copied most of this stuff from your comment in #442 🤣